### ISSUE TITLE
With poor internet and before site loads, url does not show in search bar and isn't selectable

### ISSUE THREAD

**Issue by BenjaminMichaelis** [brave/brave-ios#5574]
### Description: 
If the internet is poor or just slow (so it is loading for the first time) from a clicked hyperlink to a new tab, the url does not show.

### Steps to Reproduce 
  1.Click on a hyperlink so it opens up a new tab w/ slow internet
  2. Observe the url not being in the search bar

**Actual result:**

![image](https://user-images.githubusercontent.com/22186029/175348695-b71812fb-b6f7-45e5-8960-37033d483fbd.png)

https://user-images.githubusercontent.com/22186029/175348819-d24ca51f-535f-475f-9175-38cd8042b23a.MOV

**Expected result:**
For the url to be able to be seen and selected/modified even as it is loading for the first time, not only once it has loaded, like the page in the example tab in the video that is loaded already, except obviously while loading the page would just be blank and still have the x to stop loading.

**Reproduces how often:** [Easily reproduced, Intermittent Issue]

Easily reproduced when internet is slow/poor

**Brave Version:**

Version 1.39 (22.6.3.14) BraveCore 1.39.111 (102.0.5005.61) Device iPhone14,2 (iOS 15.5)

**Device details:** 

iPhone 13 Pro, ios 15.5

### Additional Information

**Comment by BenjaminMichaelis** [comment]
Currently once internet improves, the page loads and then the x turns into the refresh button on the right like normal, and then shows the url, like in this example you can see the url not displayed at first (but the url is known already as the page is being loaded of course) and only displays after loading.

https://user-images.githubusercontent.com/22186029/175548725-1c73c457-b4af-4b57-9c67-33a2b8c898ba.MOV

**Comment by iccub** [comment]
I find it annoying too, would be nice to make it better

**Comment by iccub** [comment]
Kyle fixed it in another ticket
